apps/system/locales/system.en-US.properties

```diff
@@ -516,6 +516,9 @@ icc-message-maintitle=Operator message
 icc-message-subtitle=SIM {{id}}
 icc-confirmCall-defaultmessage=Call {{number}}?
 icc-alertMessage-defaultmessage=Sending message
```

Comment:
Check if this message (icc-alertMessage-defaultmessage) is used somewhere, if not used anywhere, please remove it